worlds/AutoWorld.py

```diff
@@ -498,12 +498,24 @@ def create_filler(self) -> "Item":
     def get_location(self, location_name: str) -> "Location":
         return self.multiworld.get_location(location_name, self.player)
 
+    def get_locations(self) -> Iterable["Location"]:
```

Comment:
If any part of an annotation is in quotes, it can save time on the evaluation of the expression if if all of it is in quotes.

```python
    def get_locations(self) -> "Iterable[Location]":
```